### ISSUE TITLE
error cb() never called in CI on Build Server

### ISSUE THREAD

**Issue by yasiryc** [npm/npm#20993]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [#] npm is producing incorrect or undesirable behavior.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?

> npm install command is failing with following error
npm ERR! cb() never called!
npm ERR! This is an error with npm itself. Please report this error at:

#### How can the CLI team reproduce the problem?

When using npm to install angular modules during CI, i get the error. This used to work fine but suddenly stopped working. No change in terms of npm or node version. Here is the log from log file

18539 silly extract caniuse-db@1.0.30000820 extracted to D:\BuildAgentWork\c6e1dbcfcdf66d52\WebUI\NgUI2\node_modules\.staging\caniuse-db-474bdf68 (4899ms)
18540 silly extract es5-ext@0.10.41 extracted to D:\BuildAgentWork\c6e1dbcfcdf66d52\WebUI\NgUI2\node_modules\.staging\es5-ext-1cd39abf (8843ms)
18541 silly extract core-js@2.3.0 extracted to D:\BuildAgentWork\c6e1dbcfcdf66d52\WebUI\NgUI2\node_modules\.staging\core-js-4aeb49fc (7854ms)
18542 silly extract core-js@2.5.4 extracted to D:\BuildAgentWork\c6e1dbcfcdf66d52\WebUI\NgUI2\node_modules\.staging\core-js-30936494 (6008ms)
18543 silly extract @angular/common@5.2.9 extracted to D:\BuildAgentWork\c6e1dbcfcdf66d52\WebUI\NgUI2\node_modules\.staging\@angular\common-03fec8bb (21757ms)
18544 silly extract fsevents@1.1.3 extracted to D:\BuildAgentWork\c6e1dbcfcdf66d52\WebUI\NgUI2\node_modules\.staging\fsevents-113ac709 (7694ms)
18545 silly extract rxjs@5.5.8 extracted to D:\BuildAgentWork\c6e1dbcfcdf66d52\WebUI\NgUI2\node_modules\.staging\rxjs-8a928e38 (16450ms)
**18546 timing npm Completed in 45426ms
18547 error cb() never called!
18548 error This is an error with npm itself. Please report this error at:
18549 error <https://github.com/npm/npm/issues>**

### supporting information:

 - `npm -v` prints: 6.1.0
 - `node -v` prints: 8.11.2
 - `npm config get registry` prints: https://registry.npmjs.org
 - Windows, OS X/macOS, or Linux?: Windows Server 2012

**Comment by lzocateli** [comment]
I have the same problem !!

C:\agent\npm -v
6.1.0
C:\agent\node -v
v10.4.0

Buind in machine OS: Windows 10

**Comment by yasiryc** [comment]
Thanks to the information on https://github.com/npm/npm/issues/7014. I am able to resolve the issue on my build agent. In my case the issue was related to custom registry entry, We use our local registry to pull down some npm packages and our local registry entry had an extra "'". Removed it and able to build fine on build agent. CI is back to normal for now. I will reopen the ticket if issue comes back. 

Just quick summary what i did,

I enabled the detailed log by using " npm -dd install --no-package-json" and it showed my custom package didn't have a valid url to pull down. Fixed it and reverted back to "npm -dd install" and it builds fine.
